lib/src/interfaces/Options.ts

```diff
@@ -1240,6 +1247,11 @@ setRoot: {
    * #### (iOS specific)
    */
   rootBackgroundImage?: ImageResource;
+  /**
+   * Background color for the screen
+   * #### (iOS specific)
+   */
```

Comment:
The comment should be more generalised as `WindowOptions` can potentially include more than just the background color.